sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/protocol/share_rest_client.hpp

```diff
@@ -1772,11 +1774,7 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
           else if (node.Type == XmlNodeType::StartTag)
           {
 
-            if (std::strcmp(node.Name, "SMB") == 0)
```

Comment:
Why don't we need to check for this anymore? Also, add tests.

Reply:
Previously there was a bug adding a redundant `SMB` tag parsing, which is already done in `ShareProtocolSettingsFromXml`.

Reply:
This is again covered in `DISABLED_SetPremiumFileProperties`. This test is manually run against a test environment but disabled due to production not ready.